eng/versioning/version_data.txt

```diff
@@ -46,3 +46,4 @@ com.microsoft.azure.msi_auth_token_provider:azure-authentication-msi-token-provi
 com.microsoft.azure:azure-eventgrid;1.4.0-beta.1;1.4.0-beta.1
 com.microsoft.azure:azure-loganalytics;1.0.0-beta-2;1.0.0-beta.2
 com.microsoft.azure:azure-media;1.0.0-beta.1;1.0.0-beta.1
+
```

Comment:
nit: why is this file even in the PR, there are no changes and we don't need another version because of a whitespace change.

Reply:
Got it. Removed the unnecessary changes.